messages_appendix.cddl

```diff
@@ -66,12 +66,18 @@ psk-input-method = &(
   qr-code: 1
 )
 
+auth-initiation-password = (
+  ? 0 : text .size 24 ; password
+)
+
 ; type key 1002
 auth-spake2-need-psk = {
+  auth-initiation-password
 }
 
 ; type key 1003
 auth-spake2-message = {
+  auth-initation-password
```

Comment:
Is it needed for this message?  It seems like it's just needed for auth-spake2-need-psk.

Reply:
It's needed when the PSK presenter initiates authentication.